### DISCUSSION TITLE
Giving RubyGems access

### DISCUSSION THREAD

**Discussion by benjaminoakes** [benjaminoakes/maid#196] [General]
@coaxial I'm trying to make a release of v0.9.0.alpha.1 to see if I can even do that anymore.  After that, I'd like to also give you access.  Do you have any thoughts on how we might give you access to https://rubygems.org/gems/maid?

**Comment by coaxial** [comment]
I'm reading the docs to try and find out, because I've never done that.

**Comment by coaxial** [comment]
I think this is how: https://guides.rubygems.org/managing-owners-using-ui/ (section "Adding user as an owner to your gem")

**Comment by benjaminoakes** [comment]
![Screenshot from 2023-03-18 11-54-12](https://user-images.githubusercontent.com/5323/226121311-51803f46-b016-41a1-8443-d9af319b26aa.png)


@coaxial Can you please share the RubyGems handle that you would like to use?

**Reply by coaxial** [reply]
I'm also `coaxial` on there.

**Reply by benjaminoakes** [reply]
Cool.  I'd like to make sure the docs for how to release are up to date.  At some point, it might be nice to automate the release with CI.  (I've done that on other projects and it's quite nice to have.)

**Reply by coaxial** [reply]
> automate the release with CI

Definitely

**Reply by benjaminoakes** [reply]
Released!  https://rubygems.org/gems/maid/versions/0.9.0.alpha.1

**Reply by benjaminoakes** [reply]
Properly tagged... https://github.com/maid/maid/releases/tag/v0.9.0.alpha.1

**Reply by benjaminoakes** [reply]
Below is a transcript.  This was all that was necessary.  It probably deserves to be documented somewhere, but I couldn't find if I had previously documented it.  

(I happened to run this on a GitHub Codespace, in case that's helpful for reproducing the steps.)

```
@benjaminoakes ➜ /workspaces/maid (master) $ rake release
maid 0.9.0.alpha.1 built to pkg/maid-0.9.0.alpha.1.gem.
Tagged v0.9.0.alpha.1.
Pushed git commits and release tag.
Enter your RubyGems.org credentials.
Don't have an account yet? Create one at https://rubygems.org/sign_up
   Email:   benjaminoakes
Password:   

You have enabled multi-factor authentication. Please enter OTP code.
Code:   [REDACTED]
Signed in with API key: [REDACTED]
Pushing gem to https://rubygems.org...
Successfully registered gem: maid (0.9.0.alpha.1)
Pushed maid 0.9.0.alpha.1 to rubygems.org
```

**Reply by benjaminoakes** [reply]
> coaxial was added as an unconfirmed owner. Ownership access will be enabled after the user clicks on the confirmation mail sent to their email.

**Reply by coaxial** [reply]
Done, it's official!

**Reply by benjaminoakes** [reply]
@coaxial At this point, you should be able to make a release.  If you'd like to try that now: https://github.com/maid/maid/pull/197#issuecomment-1474916821

**Reply by coaxial** [reply]
I have to step away right now, but I'll do that when I have the time; probably tomorrow. I'm on GMT+1 and it's dinner time here :)

**Reply by benjaminoakes** [reply]
You're doing great stuff so far.  Thank you again!  It was nice to see high-quality contributions show up today.  I was also impressed by what I could find about you online.  I had been thinking about finding another maintainer for Maid, so it's great that you reached out.  As I said, I still intend to be a part of this project, but realistically, my attention is needed elsewhere most days.  My contributions will be limited.  I still care about this project though, so I really appreciate your help!

I need to go do some other things too, but at this point, you should have everything you need to take this further!  I'd encourage discussions like this one about any major changes in direction, but I hope to generally stay on the sidelines and cheer you on.  :smile:

**Comment by coaxial** [comment]
I was able to release `0.9.0.alpha.2`. I'm looking into automating this as my next task.

**Reply by benjaminoakes** [reply]
Great! I'm sure there's some GitHub Action for it. Does it make sense to put some (more?) branch protection rules for master in place too?

**Reply by coaxial** [reply]
I think it should be ok the way it is now.

**Reply by benjaminoakes** [reply]
Agreed! I wrote this before I saw what you implemented
